src/pages/Login.tsx

```diff
@@ -0,0 +1,108 @@
+import { Form, Field } from "react-final-form";
+
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const Login = () => {
+  const user: { email: string; password: string } = {
+    email: "pathomphob.s@20scoops.net",
+    password: "20scoop",
+  };
+
+  const onSubmit = (values: { email: string; password: string }) => {
+    if (values.email === user.email && values.password === user.password) {
+      toast.success("Success Login!", { autoClose: 3000 });
+    } else {
+      toast.error("Email or Password is incorrect", { autoClose: 3000 });
+    }
+  };
+
+  const validateEmail = (email?: string) => {
+    return String(email)
+      .toLowerCase()
+      .match(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      );
+  };
+
+  return (
+    <div className="w-screen h-screen">
+      <div className="w-screen h-screen flex flex-col justify-center bg-black">
+        <Form
+          onSubmit={onSubmit}
+          validate={(values) => {
+            const errors: { email?: string; password?: string } = {};
+            console.log(errors);
+            if (!validateEmail(values.email)) {
+              errors.email = "Email is not valid";
+            }
+            if (!values.email) {
+              errors.email = "Email is required";
+            }
+
+            if (!values.password) {
+              errors.password = "Password is required";
+            }
+
+            return errors;
+          }}
+          render={({ handleSubmit, values, errors }) => (
+            <form
+              onSubmit={handleSubmit}
+              className="mx-auto w-11/12 sm:w-7/12 lg:w-4/12 flex flex-col shadow-lg p-8 md:p-10 bg-white rounded ease-out duration-300"
+            >
+              <p className="text-center uppercase font-bold pb-8 text-xl">
+                Login
+              </p>
+
+              <label className="text-md py-2 font-light">Email</label>
+              <Field name="email">
+                {({ input, meta }) => (
+                  <div className="flex flex-col">
+                    <input
+                      {...input}
+                      className="h-12 px-4 w-100 bg-gray-100 focus:bg-gray-200 rounded focus:outline-none ring-black focus:ring-2"
+                      type="text"
+                      placeholder="Email"
+                    />
+                    {meta.error && meta.touched && (
+                      <span className="text-xs text-red-500 pt-1">
+                        {meta.error}
+                      </span>
+                    )}
+                  </div>
+                )}
```

Comment:
@bbun0639 could we separate to another component so we reduce duplicate for email field and password field ?